mesonbuild/environment.py

```diff
@@ -934,6 +938,7 @@ def detect_cpp_compiler(self, for_machine):
     def detect_cuda_compiler(self, for_machine):
         popen_exceptions = {}
         compilers, ccache, exe_wrap = self._get_compilers('cuda', for_machine)
+        info = self.machines[for_machine]
```

Comment:
__[Flake8]__ [F841] local variable 'info' is assigned to but never used ([link](https://sider.review/gh/repos/19784232/pulls/5833#issue-4550409))

<sub>posted by Sider</sub>